functions/localport_freebsd.go

```diff
@@ -1,82 +1,50 @@
-//go:build freebsd
-// +build freebsd
+//go:build freebsd || darwin
```

Comment:
should the dawrin functions be in the localport_freebsd.go file? or maybe it should be renamed

Reply:
sgould be _unix with !linux